packages/accessibility/src/AccessibilityManager.ts

```diff
@@ -4,8 +4,8 @@ import { accessibleTarget } from './accessibleTarget';
 
 import type { Rectangle } from '@pixi/math';
 import type { Container } from '@pixi/display';
-import type { Renderer, AbstractRenderer } from '@pixi/core';
 import type { IAccessibleHTMLElement } from './accessibleTarget';
+import { IRenderer } from 'packages/core/src/IRenderer';
```

Comment:
Please be mindful that VSCode doesn't auto-import correctly for monorepo. You'll need to fix this in a few places it seems.
```suggestion
import type { IRenderer } from '@pixi/core';
```